astrocut/asdf_cutouts.py

```diff
@@ -38,7 +45,7 @@ def _get_cloud_http(s3_uri: Union[str, S3Path], verbose: bool = False) -> str:
         print(f'Attempting to access private S3 bucket: {s3_path.bucket}')
 
     # create file system and get URL of file
-    fs = s3fs.S3FileSystem(anon=is_anon)
+    fs = s3fs.S3FileSystem(anon=is_anon, key=key, secret=secret, token=token)
```

Comment:
Do we want a check here to make sure one of the access inputs is set when `is_anon` is False?

Reply:
For certain cases, a credential may not be needed. For instance, RSP doesn't need to pass credentials because the environment has direct access to the private S3 bucket.

Reply:
that makes sense, thanks!